### PR TITLE
Vue training - 03 - API

### PR DESCRIPTION
**Pull request by rsuescun**
## Summary

-Get data from api
-Mocked api with `apisauce` and `json-server` in `http://localhost:3002/techs`

**Note:** The api was simulated because the suggested endpoint https://demo6061045.mockable.io/wolox/techs returned a`404`

## Screenshots

Screen Shot 2019-04-23 at 10 54 58 AM

## Trello Card

https://trello.com/c/SsAkTURB/15-vue-components

### CODE REVIEW

File: vue-training/json-server.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "port": 3002
```
Comment (FrankIglesias) [inline vue-training/json-server.json]: Why this port?

File: vue-training/src/services/api.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { create } from 'apisauce'
+
+const api = create({
+  baseURL: 'http://localhost:3002'
```
Comment (FrankIglesias) [inline vue-training/src/services/api.js]: use a .env

File: vue-training/src/views/index/index.js [hunk 3]
```diff
@@ -2,86 +2,29 @@ import Vue from 'vue'
...
-          }
-        ]
-      }
-    ]
+    techs: []
+  },
+  created() {
+    TechService.getTechs()
+      .then(response => {
+        vm.techs = response.data
```
Comment (FrankIglesias) [inline vue-training/src/views/index/index.js]: why not this?

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: vue-training/json-server.json (hunk 1 above)
Comment (rsuescun) [reply vue-training/json-server.json]: I forgot to delete it

File: vue-training/src/views/index/index.js (hunk 3 above)
Comment (rsuescun) [reply vue-training/src/views/index/index.js]: I thought it was a good idea to use 'vm' in the creation and thus avoid disabling the eslint rule 'no-unused-vars'

File: vue-training/src/services/api.js (hunk 2 above)
Comment (rsuescun) [reply vue-training/src/services/api.js]: Added `API_URL='http://localhost:3002` in `.env` file

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**